CommonMark.Console/Program.cs

```diff
@@ -140,6 +171,34 @@ static int Main(string[] args)
                     }
                 }
 
+                if (useFatHtmlFormatter)
+                {
+                    settings.OutputFormat = OutputFormat.CustomDelegate;
+
+                    if (placeholderSubstitutionTable == null)
+                    {
+                        settings.OutputDelegate = (block, writer, cmSettings) =>
+                        {
+                            var formatter = new Formatters.HtmlFormatter(writer, cmSettings);
+                            formatter.WriteDocument(block);
+                        };
+                    }
+                    else
+                    {
+                        settings.OutputDelegate = (block, writer, cmSettings) =>
+                        {
+                            var formatter = new Formatters.HtmlFormatter(writer, cmSettings);
+                            formatter.PlaceholderResolver = placeholderText =>
```

Comment:
I noticed if a benchmark is run, this distinction would be within the benchmarked code, but as that particular instruction should be hit only once, I was probably overoptimizing. Changed.